test/cmdlineTests/standard_no_append_cbor/output.json

```
@@ -9,7 +9,23 @@
                         "linkReferences": {},
                         "object": "<BYTECODE REMOVED>",
                         "opcodes":"<OPCODES REMOVED>",
-                        "sourceMap":"<SOURCEMAP REMOVED>"
+                        "sourceMap":"<SOURCEMAP REMOVED>",
+                        "subAssemblyOffsets": {
+                            "subs": [
+                                {
+                                    "isCreation": true,
+                                    "length": 27,
+                                    "start": 0,
+                                    "subs": [
```

The top-level assembly is not nested in anything. Seems weird to put its data under `'subs'`. It should be at the top level. Similarly with `subAssemblyOffsets` - it should rather be `assemblyOffsets` since it covers the whole binary, including the top.

Finally, we might end up including more info there later so maybe `assemblyStructure` would be a more future-proof name?

```suggestion
                        "assemblyStructure": {
                            "isCreation": true,
                             "length": 27,
                             "start": 0,
                             "subassemblies": [
```

I'd use that name for `LinkerObject::SubAssembly` too, i.e. `LinkerObject::Structure`, which would nicely mirror `Object::Structure` which is similar in the sense that it stores metadata about the hierarchy (but on Yul level).